webpack.config.js

```diff
@@ -199,7 +199,7 @@ module.exports = [
         entry: {
             'alwaysontop': './react/features/always-on-top/index.js'
         },
-        performance: getPerformanceHints(400 * 1024)
+        performance: getPerformanceHints(500 * 1024)
```

Comment:
Why you changed the value here, I don't see any relation between ToolboxItem?

Reply:
Yeah that's why you got a lint error message, something went wrong. From my perspective it has a purpose with specific value. Please validate/debug why.

Reply:
The tooltip change has the ID of alwaysOnTop which then renders the tooltip as alwaysOnTop. There is no need to validate/debug this. A slight increase of this file does not affect the app's performance and has no side effect or what so ever.